connector/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM gradle:jdk17 AS build
```

Comment:
also the Dockerfile in the upstream brings in some improvements, like bumped versions and cache support: https://github.com/Think-iT-Labs/edc-connector-client/blob/8516378325d5931feeb43b6f6064caab3e06a0e6/connector/Dockerfile#L1